### DISCUSSION TITLE
Windows container unable to mount untill run memfsx64 mount in hosting node

### DISCUSSION THREAD

**Discussion by AsifImam** [winfsp/winfsp#552] [Q&A] [answered]
Hi all ,
Despite I can see winfsp.launcher service running in the host, the container is unable to mount any drive using rclone. As soon as I run memfs inside host, and even just kill it later , next moment i am able to mount a drive.
It looks like host needs some dlls to be loaded and winfsp.launcer seem not helping. 
Any hint ? otherwise I'll have to modify base node's image. thanks.

**Comment by billziss-gh** [comment, marked as answer]
Sorry for the late response.

This is by design. For a WinFsp file system to work, the WinFsp driver needs to be loaded. When the driver is not loaded a WinFsp file system can start the driver when it is launched outside a container. But inside a Windows container it is not possible to start file system drivers.

**Reply by AsifImam** [reply]
Ahh .. ok ..Got it. 
You should dedicate this task to launcher service or a flag in launcher service to start driver. (for future).
Thanks for the clarification I am clear about my understanding now.
